### ISSUE TITLE
Don't treat entry headers that end with `*` as author lines

### ISSUE THREAD

**Issue by zzak** [skipkayhil/rails-bin#3]
Originally reported here:
https://github.com/rails/rails/pull/41994#issuecomment-1478544614

For example,

```
*   Expand search field on `rails/info/routes` to also search **route name**, **http verb** and **controller#action**

    *Jason Kotchoff*
```

Since the title ends with `*` we assume its a author line, such as `*Full Name*` which must be indented.

**Comment by zzak** [comment]
Also weird this was only failing on PRs but not on main 🤔

**Comment by skipkayhil** [comment]
Fixed in afa12723dedfaf0188f9da1653daf3eb163114ca
